src/scripts/main.js

```
@@ -354,7 +354,22 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
```

Critical: add a defensive null-check after selecting the table. Right now the code does `const dashboard = document.querySelector('.dashboard');` and immediately uses it which will throw a runtime error if the selector returns `null` (tests or a changed HTML could cause this). Add an explicit check and fail early with a clear message, for example:

```javascript
const dashboard = document.querySelector('.dashboard');
if (!dashboard) {
  throw new Error("Table with class 'dashboard' not found");
}
```

Place this check immediately after the querySelector to make failures explicit and easier to diagnose. (This is required to avoid runtime exceptions during tests.)